packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -263,6 +263,74 @@ void main() {
         expect(scrollView.controller, primary2);
       });
 
+      testWidgets('Test custom ScrollController behavior when set', (WidgetTester tester) async {
+        const Key firstBox = Key('C');
+        const Key secondBox = Key('B');
+        const Key thirdBox = Key('A');
+        
+        final ScrollController customController = ScrollController();
+
+        await tester.pumpWidget(
+          MaterialApp(
+            home: Scaffold(
+              body: SizedBox(
+                height: 200,
+                child: ReorderableListView(
+                  scrollController: customController,
+                  children: const <Widget>[
```

Comment:
nit: prefer to have the `child` or `children` property to be the last property specified for `Widget`s. ie.
```dart
Row(
  // all properties here
  // ...
  children: [
    // children widgets here
  ],
)
```